cuda_bindings/docs/source/release/12.X.Y-notes.rst

```diff
@@ -27,6 +27,8 @@ Highlights
 * Python bindings in every module, including ``driver``, ``runtime``, and ``nvrtc``, now have the GIL
   released before calling the underlying C APIs.
 
+* The Python overhead of calling functions in CUDA bindings in ``driver``,
+  ``runtime`` and ``nvrtc`` has been reduced by approximately 30%.
```

Comment:
Approved this PR to go in but we need to make sure to add this note in `main` into the `12.X.Y` release notes as it's only rendered from `main` and not from `12.9.x`.

Reply:
https://github.com/NVIDIA/cuda-python/pull/923